### PR TITLE
Update tracks, migrate dev to main

### PR DESCRIPTION
**Pull request by veidenberg**
### Description
This PR:
- Migrates from dev to main branch (to remove dev branch)
- Updates ci to migrate from dev to main branch
- Brings all beta 3 track data updates to main branch for reloading the tracks database
  - redistribution of track order values ([this commit](https://github.com/Ensembl/ensembl-web-track-api/commit/fb39649948b7827e3070e035670a7af722d31cb6); visible track order not changed)
  - will go together with already merged changes ([this](https://github.com/Ensembl/ensembl-web-track-api/commit/c6b3dcf10d412f9be38aaed81ce0ab3e69ca4e7e) and [this](https://github.com/Ensembl/ensembl-web-track-api/commit/5bf33362e2ad3aadd9261ccfe3e7be142f8d6cb9)) to update tracks DB
- Updates documentation

### Related JIRA Issue(s)
[ENSWBSITES-2840](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2840)

### Review App URL(s) 
Tracks have been reloaded from this branch to dev: http://dev-2020.enseml.org

### Knowledge Base
NA

### Example(s)
Track payloads from Track API should have the metadata values (e.g. labels/descriptions) included in this PR (templates/csv files).

### Related tasks
After this PR is merged:
* Reload tracks to staging/prod (SOP) after tracks look good in dev
* Remove dev branch after the new ci/cd pipeline is confirmed working

### Checklist

- [ ] Black formatting
- [ ] Tests

### CODE REVIEW

File: utils/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Examples:
+```bash
+export TRACK_API_URL=http://localhost:8000 # target track API
+export TRACK_DATA_DIR=/Users/Alice/datafiles # track datafiles
+
+```
+
+### Legacy scripts
+
```
Comment (azangru) [inline utils/README.md:19]: What happens here?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -1,102 +1,101 @@
...
+        - kustomize edit set image DOCKER_IMAGE=${DOCKER_IMAGE}
         - kubectl apply -k .
 
 .deploy-main:
     extends:
         - .deploy
         - .branch-rules-main
 
-.deploy-dev:
+.deploy-all:
```
Comment (azangru) [inline .gitlab-ci.yml:38]: "all" meaning "other than main"? I got confused here for a minute, thinking that it means "deploy everything". Perhaps `.deploy-review` or `.deploy-non-main` would better reflect the purpose of this template?

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -1,102 +1,101 @@
...
   environment:
     name: staging
   variables:
     AGENT: ${STAGING_AGENT}
     NS: ${STAGING_NS}
 
 Internal:
-  extends: .deploy-dev
+  stage: deploy-preview
+  extends: .deploy-all
```
Comment (azangru) [inline .gitlab-ci.yml:89]: Why does it not deploy from main?

More generally, do we still use internal?

File: utils/README.md (hunk 1 above)
Comment (veidenberg) [reply utils/README.md:19]: Added missing content

File: .gitlab-ci.yml (hunk 3 above)
Comment (veidenberg) [reply .gitlab-ci.yml]: Removed

File: .gitlab-ci.yml (hunk 2 above)
Comment (veidenberg) [reply .gitlab-ci.yml]: Renamed

**Review by azangru — APPROVED** [review]

**Pull request merged**